.gitignore

```diff
@@ -16,3 +16,4 @@ __pycache__
 
 # Jupyter checkpoints
 **/.ipynb_checkpoints/*
+recipes/redlionfish/collect_ssh256_key.bat
```

Comment:
Let's not do this, this .gitignore is used by every recipe ever submitted, but this recipe directory will be removed when it's converted to a feedstock.

Reply:
oh dear. I can't get it right. First time trying to create a conda-forge package...

Reply:
No worries! Learning new things is challenging.